packages/sanity/src/core/form/studio/inputs/client-adapters/reference.ts

```diff
@@ -28,6 +40,7 @@ const NOT_FOUND = {
   reason: 'NOT_FOUND',
 } as const
 
+// type VersionsPreviewTuple = [bundleId: string, snapshot: PreviewValue | null]
```

Comment:
Seems to not be used anywhere, perhaps we can just remove it? :)

Reply:
absolutely, thanks for catching that!

Reply:
will remove in next push